Terminal.Gui/Views/Label.cs

```diff
@@ -21,19 +21,13 @@ public class Label : View {
 	/// <inheritdoc/>
 	public Label (string text, bool autosize = true) : base (text) => SetInitialProperties (autosize);
```

Comment:
I'm seeing you didn't remove the `text` constructors. Am I missing a reason why they shouldn't be removed as well?

It's easy enough to do:

```cs
var l = new Label() { Text = "text", AutoSize = false }
```

Reply:
I now realize you want get rid all the parameters and I'll fix it.